index.html

```diff
@@ -1192,33 +1192,31 @@ <h3>
       </p>
 
       <p>
-        A DID method specification should have a section dedicated to personal
-        data covering the extent to
+        DID method specifications are encouraged to have a section dedicated to
+        personal data, covering the extent to
         which information published on the corresponding ledger can be updated
         or deleted.
-        It should provide specific instructions of how to do so, if it is
-        possible.
+        Such a section could provide specific instructions of how to do so,
+        if it is possible.
         Otherwise, it should clearly state that it is not possible.
       </p>
 
       <p>
         A DID method should avoid "phone home" or tracking characteristics that
-        would
-        permit tracing of a user in manner not understood or authorized by the
-        user
-        by some third party.
+        would permit tracing of a user by a third party
```

Comment:
```suggestion
        would enable tracking of a user, without consent, by a third party
```

Reply:
Change committed in 26b848ba9e49008e1fc8860b08a46f4fc6404eaa

Reply:
Discussing further in https://github.com/w3c/did-imp-guide/pull/37#discussion_r762321582